jerry-core/parser/js/parser.cpp

```diff
@@ -830,6 +830,14 @@ parse_primary_expression (void)
 
   switch (tok.type)
   {
+    case TOK_DIV:
+    case TOK_DIV_EQ:
+    {
+      // must be a regexp literal so re-lex token as a regexp literal
+      lexer_seek (tok.loc);
+      skip_token (true);
+      /* FALLTHRU */
```

Comment:
I would prefer to add a function:
```
static void
rescan_regexp_token ()
{
  lexer_seek (tok.loc);
  tok = lexer_next_token (true);
}
```
and call it here.  This would allow removing default argument from `skip_token` and make the code more explicit.